python/run-tests.py

```diff
@@ -57,6 +57,10 @@ def print_red(text):
 FAILURE_REPORTING_LOCK = Lock()
```

Comment:
we need `#!/usr/bin/env python` -> `#!/usr/bin/env python3`?

Reply:
yeah, good idea.